ext/opcache/zend_persist.c

```diff
@@ -1048,8 +1048,9 @@ zend_class_entry *zend_persist_class_entry(zend_class_entry *orig_ce)
 			for (i = 0; i < ce->num_interfaces; i++) {
 				zend_accel_store_interned_string(ce->interface_names[i].name);
 				zend_accel_store_interned_string(ce->interface_names[i].lc_name);
+				// TODO: should we store .is_optional here? How?
```

Comment:
No special handling is needed, the bool is copied by `zend_shared_memdup_free` down below.

Reply:
Thanks for the explanations!